sql/analyzer/indexed_joins.go

```diff
@@ -721,6 +721,12 @@ func addCrossHashJoins(m *memo.Memo) error {
 			return nil
 		}
 
+		_, lIsSqa := join.Left.First.(*memo.SubqueryAlias)
```

Comment:
there might be filters or projections in-between. you'll probably want to check the list of TableIdNodes in each group for a single SQA